src/main/scala/microsites/MicrositeKeys.scala

```diff
@@ -133,6 +133,10 @@ trait MicrositeKeys {
     "Optional. Add custom Gitter sidecar Chat URL. By default is owner/repository."
   )
 
+  val micrositeFooterText: SettingKey[Option[String]] = SettingKey[Option[String]](
```

Comment:
Would this make more sense as `Option[TypedTag[String]]` for more type safety?

Reply:
you mean as as `scalatags` tag? I think is fine like that but if scalatags types make sense here then sure.

Reply:
Yup, `scalatags`. My only apprehension is would the user need to add another dependency? Or does the `scalatags` dependency come for "free" when a user adds `sbt-microsites`?

The current `String` allows one to always `{blah: TypedTag[String]}.toString`.